@xen-orchestra/web/src/types/xo/alarm.type.ts

```diff
@@ -0,0 +1,15 @@
+import type { Branded, XapiXoRecord } from '@vates/types'
+
+export type XoAlarm = {
```

Comment:
To avoid rewrite your type, use `Pick`.

```ts
import type {XoAlarm as VatesXoAlarm} from '@vates/types'

export type XoAlarm = Pick<VatesXoAlarm, 'time' | 'body' | 'object'> & {
 id: Branded<'ALARM'>
 }
```